### PR TITLE
API to search destination attributes on value column using icontains operator

### PR DESCRIPTION
**Pull request by Sravanksk**

### CODE REVIEW

File: fyle_accounting_mappings/views.py [hunk 1]
```diff
@@ -133,3 +133,20 @@ def get_queryset(self):
...
+    """
+    serializer_class = DestinationAttributeSerializer
+
+    def get_queryset(self):
+        destination_attribute_value = self.request.query_params.get('destination_attribute_value')
+
+        assert_valid(destination_attribute_value is not None, 'query param destination_attribute_value not found')
+
+        destination_attributes = DestinationAttribute.objects.filter(
+            value__icontains=destination_attribute_value, workspace_id=self.kwargs['workspace_id']
```
Comment (ashwin1111) [inline fyle_accounting_mappings/views.py:150]: filter should be on attribute_type as well, plis to expect this from client

File: fyle_accounting_mappings/views.py [hunk 2]
```diff
@@ -133,3 +133,20 @@ def get_queryset(self):
...
+        ).all()
+        return destination_attributes
```
Comment (ashwin1111) [inline fyle_accounting_mappings/views.py:152]: Will this be in a paginated fashion? may be, let's add a hard limit to it, like 50? thoughts?

File: fyle_accounting_mappings/views.py (hunk 1 above)
Comment (Shwetabhk) [reply fyle_accounting_mappings/views.py:150]: +1

Comment (Sravanksk) [reply fyle_accounting_mappings/views.py:150]: 👍

File: fyle_accounting_mappings/views.py (hunk 2 above)
Comment (Shwetabhk) [reply fyle_accounting_mappings/views.py:152]: I think list views should be paginated by default, limit can be set from the UI right @ashwin1111 ?

**Review by Shwetabhk — APPROVED** [review]

File: fyle_accounting_mappings/views.py (hunk 2 above)
Comment (Sravanksk) [reply fyle_accounting_mappings/views.py:152]: nope, you don't get a paginated response

Comment (Shwetabhk) [reply fyle_accounting_mappings/views.py:152]: @Sravanksk don't we have a pagination class in settings.py?

Comment (Shwetabhk) [reply fyle_accounting_mappings/views.py:152]: I think it automatically paginates all the List APIs
Please confirm

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**